outputs.tf

```diff
@@ -12,3 +20,40 @@ output "lifecycle_policy" {
   description = "The attached repository lifecycle policies."
   value       = try(aws_ecr_lifecycle_policy.lifecycle_policy[0], null)
 }
+
+# ------------------------------------------------------------------------------
+# OUTPUT ALL INPUT VARIABLES
+# -----------------------------------------------------------------------------
+
+output "module_inputs" {
+  description = "A map of all module arguments. Omitted optional arguments will be represented with their actual defaults."
+  value = {
+    name                         = var.name
+    immutable                    = var.immutable
+    scan_on_push                 = var.scan_on_push
+    repository_policy_statements = var.repository_policy_statements
```

Comment:
this will be computed and include the statements for push and pull in the end. so this needs to be adjusted...

we never used module_inputs yet and i am not sure if it is worth the effort to calculate outputs just for doing it.. i know it is in the template but it is a stale feature.

the overhead to maintain this seems too big compared to not having a real use case.

Reply:
removed `module_inputs`